docs/EARLY-RELEASE.md

```diff
@@ -28,9 +29,10 @@ big and we never release just a patch. There is only "release".
  - Is there a security advisory rated high or critical?
```

Comment:
A high security advisory doesn't necessarily mean an immediate patch release unless the flaw is publicly known. But, I suppose it is "a question to ask".